src/io/std_file_descriptor.cr

```diff
@@ -0,0 +1,80 @@
+require "../signal"
+require "c/sys/time"
+require "c/time"
+
+class IO::StdFileDescriptor < IO::FileDescriptor
```

Comment:
Please `:nodoc:` this actually.

Reply:
Maybe even push it under Crystal::System?

Reply:
Yeah I like that idea.